tl-dev-1.rockspec

```diff
@@ -16,6 +16,9 @@ dependencies = {
 
    -- needed for the cli tool
    "argparse"
```

Comment:
There's a missing comma here (@hishamhm, I think CI should've caught this?)

Reply:
oh good catch, that's right

Pushing #178 to catch this